<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StorageFrame</class>
 <widget class="QWidget" name="StorageFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="NewStorageButton">
     <property name="text">
      <string>New...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="StoragesTableView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
   <action name="RemoveAction">
     <property name="text">
       <string>Remove</string>
     </property>
     <property name="toolTip">
       <string>Remove selected</string>
     </property>
   </action>
   <action name="SetAction">
     <property name="text">
       <string>Edit...</string>
     </property>
     <property name="toolTip">
       <string>Edit parameters</string>
     </property>
   </action>
 </widget>
 <resources/>
 <connections/>
</ui>
